### PR TITLE
Add documentation how to use max_hops plugin

### PR DESCRIPTION
**Pull request by nick-stroud**
In the future it will be possible to include this configuration in a blueprint but for now it is blocked by upstream changes. 

This change simply documents the manual process to use `max-distance` placement flag.

### Submission Checklist

Please take the following actions before submitting this pull request.

* Fork your PR branch from the Toolkit "develop" branch (not main)
* Test all changes with pre-commit in a local branch [#](https://goo.gle/hpc-toolkit#development)
* Confirm that "make tests" passes all tests
* Add or modify unit tests to cover code changes
* Ensure that unit test coverage remains above 80%
* Update all applicable documentation
* Follow Cloud HPC Toolkit Contribution guidelines [#](https://goo.gle/hpc-toolkit-contributing)

### CODE REVIEW

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md [hunk 1]
```diff
@@ -114,6 +114,57 @@ page.
...
+force VMs to be spread across multiple racks. Google provides the `max-distance`
+flag which can used to control the maximum spreading allowed. Read more about
+`max-distance` in the
+[official docs](https://cloud.google.com/compute/docs/instances/use-compact-placement-policies
+).
+
+After deploying a Slurm cluster, you can use the following steps to manually
+configure the max-distance parameter.
+
+1. Make sure you blueprint has `enable_placement: true` on your Slurm partitions.
```
Comment (harshthakkar01) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:132]: nit: Make sure "your"

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md [hunk 2]
```diff
@@ -114,6 +114,57 @@ page.
...
+You can confirm that the `max-distance`` was applied by calling the following
+command while jobs are running:
+
+```shell
+gcloud beta compute resource-policies list \
+  --format='yaml(name,groupPlacementPolicy.maxDistance)'
+```
+
+> [!WARNING]
+> Setting `max_hops`/`max-distance` will make requests more likely to receive a
```
Comment (harshthakkar01) [inline community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md:159]: May be like: will make requests which may receive a stockout error ?

Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md]: Reworded as agreed.

**Review by harshthakkar01 — APPROVED** [review]

File: community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md (hunk 1 above)
Comment (nick-stroud) [reply community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md]: Good catch, reworded slightly as well.

**Pull request merged**